content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -87,48 +90,63 @@ Each Container of a Pod can specify one or more of the following:
 * `spec.containers[].resources.requests.memory`
 * `spec.containers[].resources.requests.hugepages-<size>`
 
-Although requests and limits can only be specified on individual Containers, it
-is convenient to talk about Pod resource requests and limits. A
-*Pod resource request/limit* for a particular resource type is the sum of the
-resource requests/limits of that type for each Container in the Pod.
+Although you can only specify requests and limits for individual containers,
+it is also useful to think about the overall resource requests and limits for
+a Pod.
+For a particular resource, a *Pod resource request/limit* is the sum of the
+resource requests/limits of that type for each container in the Pod.
 
 ## Resource units in Kubernetes
 
-### Meaning of CPU
+### CPU resource units {#meaning-of-cpu}
 
 Limits and requests for CPU resources are measured in *cpu* units.
-One cpu, in Kubernetes, is equivalent to **1 vCPU/Core** for cloud providers and **1 hyperthread** on bare-metal Intel processors.
+In Kubernetes, 1 CPU unit is equivalent to **1 physical CPU core**,
+or **1 virtual core**, depending on whether the node is a physical host
+or a virtual machine running inside a physical machine.
 
-Fractional requests are allowed. A Container with
+Fractional requests are allowed. A container with
 `spec.containers[].resources.requests.cpu` of `0.5` is guaranteed half as much
-CPU as one that asks for 1 CPU. The expression `0.1` is equivalent to the
-expression `100m`, which can be read as "one hundred millicpu". Some people say
-"one hundred millicores", and this is understood to mean the same thing. A
-request with a decimal point, like `0.1`, is converted to `100m` by the API, and
-precision finer than `1m` is not allowed. For this reason, the form `100m` might
-be preferred.
+CPU as one that asks for `1` cpu.
+The [quantity](/docs/reference/kubernetes-api/common-definitions/quantity/) expression
+`0.1` is equivalent to the expression `100m`.
+When you use that to specify CPU resources, you can think of that as "one hundred milliCPU";
+Some people say "one hundred millicores", which means the same thing.
+If you specify a resource request or limit with a decimal point, for example `0.1`,
+the Kubernetes API canonifies this to a milliCPU value.
+
+CPU resource is always specified as an absolute quantity, never as a relative quantity:
+`500m` CPU represents the roughly same amount of computing power whether that container
+runs on a single-core, dual-core, or 48-core machine.
 
-CPU is always requested as an absolute quantity, never as a relative quantity;
-0.1 is the same amount of CPU on a single-core, dual-core, or 48-core machine.
+{{< note >}}
+Kubernetes doesn't allow you to specify CPU resources with a precicion finer tham
+`1m`. Because of this, it's useful to specify CPU units less than `1.0` or `1000m` using
+the milliCPU form; for example, `5m` rather than `0.005`.
```

Comment:
```suggestion
`1m`. Because of this, it's useful to specify CPU units less than `1.0` (or `1000m` using
the milliCPU form); for example, `5m` rather than `0.005`.
```

This is a little unclear to me. Does this sentence mean that instead of specifying `0.005` for a CPU resource request, we should use `5m`, and they both mean the same amount?

Reply:
If you wrote `0.005` then a subsequent person might change that to `0.00025` (which is `0m`), whereas writing `5m`, that misguided change is less likely.

Reply:
(happy to consider another wording here; would prefer to get these tweaks merged)